src/App.js

```diff
@@ -1,19 +1,40 @@
 import React from 'react';
 import './App.css';
+import { TodoForm } from './Component/TodoForm';
```

Comment:
некорректное название папки
```suggestion
import { TodoForm } from './components/TodoForm';
```